composer.json

```diff
@@ -38,7 +38,6 @@
         "symfony/string": "^6.2.2",
         "symplify/easy-parallel": "^11.1",
         "symplify/rule-doc-generator-contracts": "^11.1",
-        "symplify/smart-file-system": "^11.2",
```

Comment:
do we need symfony/filesystem dependency now?

Reply:
symfony/filesystem is used in other part of code, so no need, also, already in vendor prefixed

https://github.com/rectorphp/rector/tree/569c7dc229ba5be538f24e39967a5aab3e2cb4c7/vendor/symfony/filesystem